lib/_http_incoming.js

```diff
@@ -93,21 +93,17 @@ IncomingMessage.prototype.destroy = function(error) {
 
 IncomingMessage.prototype._addHeaderLines = function(headers, n) {
   if (headers && headers.length) {
-    var raw, dest;
+    var dest;
     if (this.complete) {
-      raw = this.rawTrailers;
+      this.rawTrailers = headers;
```

Comment:
I'm not sure about this change, why was a separate array needed before? Can we assume this won't cause issues with the array being modified from the outside?